bundles/binding/org.openhab.binding.denon/README.md

```diff
@@ -1,133 +1,140 @@
-## Introduction
+# Denon Binding
 
-This binding supports Denon AV receivers. It is available from openHAB 1.7.0 onwards. It should be compatible with previous generation receivers (AVR-X2000/X3000 etc) as well as the current ones (X2100W/X3100W). 
+The openHAB Denon Binding allows interaction with Denon AV receivers. It should be compatible with previous generation receivers such as AVR-X2000/X3000, as well as current ones such as X2100W/X3100W. 
 
-The binding seems to work with some [Marantz receivers](https://groups.google.com/d/msg/openhab/bSTEfSRt0RU/rOI6uWUe7bIJ) too.
-## Configuration
+The binding also seems to work with some [Marantz receivers](https://groups.google.com/d/msg/openhab/bSTEfSRt0RU/rOI6uWUe7bIJ).
 
-```
-# denon:<instance>.<property>=value
+<!-- MarkdownTOC depth=1 -->
 
-# IP adress of the Denon receiver instance
-denon:avr2000.host=192.168.1.70
+- [Binding Configuration](#binding-configuration)
+- [Item Configuration](#item-configuration)
+- [Examples](#examples)
+- [References](#references)
 
-# Optional, set connection method for receiving updates. Can be http or telnet. 
-# Denon receivers only support one concurrent telnet connection, so use http if 
-# you have any other app using the telnet connection. Default = telnet
-#denon:avr2000.update=telnet
+<!-- /MarkdownTOC -->
 
-# Optional, this sets the refresh interval (in milliseconds) for all instances 
-# if you're using the http connection method. Default = 5000 
-#denon:refresh=5000
 
-```
-## Items
+## Binding Configuration
+
+The binding can be configured in the file `services/denon.cfg`.
+
+Multiple receivers may be defined. Each requires a unique <name> to be defined.
```

Comment:
`<name>` is being treated like an HTML tag and is disappearing when rendered.  I recommend enclosing all placeholders like this in `<backquotes>`.